main/templates/base_home.html

```diff
@@ -27,14 +27,15 @@
         </button>
         <div class="collapse navbar-collapse" id="navbarNav">
           <ul class="navbar-nav ms-auto">
+
+            {% if request.user.is_authenticated %}
             <li class="nav-item">
-              <a class="nav-link" href="/">Sign Up</a>
+              <a class="nav-link" href="{% url 'sign out' %}">Logout</a>
             </li>
-            {% if user.is_authenticated %}
+            {% else %}
             <li class="nav-item">
-              <a class="nav-link" href="/">Logout</a>
+              <a class="nav-link" href="/">Sign Up</a>
```

Comment:
I think this should be `Sign In`. If you mean that this is a registration page I think `register` might reduce the confusion.

Reply:
Changed to register.